lib/peers.go

```diff
@@ -23,8 +23,20 @@ type PeerMethods struct {
 	inst *Instance
```

Comment:
Change `inst *Instance` to `d dispatcher`. Now that only Dispatch is called, there's no need to keep a reference to the entire Instance.